content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Add Pentest Title to Jira Issue Summary"
+linkTitle: "Add Pentest Title to Jira Issue Summary"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users customize the summary field of 
+Jira issues created by their recipes to contain the title of the relevant pentest.
```

Comment:
I would emphasize changing from pentest finding `title` to pentest `title`. This is a great guide, but for less experienced users, it could be hard to notice the real difference from the `summary` field: it's still a `title` and a `tag` after all. Maybe it's just me who thinks it could be confusing.

<img width="496" alt="Screenshot 2024-11-04 at 09 08 44" src="https://github.com/user-attachments/assets/8b2d0aee-2260-41d4-b67a-9a52ba09e622">

Reply:
Updated. Thanks!